SwissTransfer/Sources/SwissTransferApp.swift

```diff
@@ -90,6 +96,28 @@ struct SwissTransferApp: App {
         }
     }
 
+    private func checkAppVersion() {
+        Task {
+            do {
```

Comment:
We open a `do` block but the `catch` block is missing.
It is important because, currently the call always fail